### PR TITLE
init

### PR DESCRIPTION
**Pull request by codygulley**
This is PR implements this package based on [bpmnlint-pack-config](https://github.com/nikku/bpmnlint-pack-config)

Tests have passed locally for this repository and verified the generated bundle file. 

Note - publishing this package to NPM is not done and I will need someone else to implement this as I do not know how.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+  "name": "dmnlint-pack-config",
+  "version": "0.7.0",
```
Comment (pinussilvestrus) [inline package.json]: Let's start with `0.0.0` here.

File: README.md [hunk 2]
```diff
@@ -1 +1,31 @@
-# dmnlint-pack-config
\ No newline at end of file
+# dmnlint-pack-config
+
+[![CI](https://github.com/bpmn-io/dmnlint-pack-config/workflows/CI/badge.svg)](https://github.com/bpmn-io/dmnlint-pack-config/actions?query=workflow%3ACI)
```
Comment (pinussilvestrus) [inline README.md:3]: That doesn't work yet as there is no GitHub action available.

Can you maybe add one [similar to this one](https://github.com/nikku/bpmnlint-pack-config/tree/master/.github/workflows)?

**Review by pinussilvestrus — CHANGES_REQUESTED** [review]
Thanks again! Let's go one step further and add CI directly to continuously test this.

Let's also update the commit message to follow our guidelines:

```
feat(project): initial commit
```

File: package.json (hunk 1 above)
Comment (codygulley) [reply package.json]: [fix(project): fix project version](https://github.com/bpmn-io/dmnlint-pack-config/pull/1/commits/f48a24f9ef533cf5ed08efe35fbd1a009989bd6c)
[f48a24f](https://github.com/bpmn-io/dmnlint-pack-config/pull/1/commits/f48a24f9ef533cf5ed08efe35fbd1a009989bd6c)

File: .github/CI.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+name: CI
```
Comment (pinussilvestrus) [inline .github/CI.yml]: I think this needs to go under `/.github/workflows`, otherwise GitHub actions won't pick it up.

File: README.md (hunk 2 above)
Comment (codygulley) [reply README.md:3]: This is my first attempt at CI configurations - i've copied the configuration from [nikku/bpmnlint-pack-config](https://github.com/nikku/bpmnlint-pack-config/tree/master/.github/workflows). 

[feat(project): add CI configuration file](https://github.com/bpmn-io/dmnlint-pack-config/pull/1/commits/90b944637a42eafd4d365701e84058d66f680d0c)
[90b9446](https://github.com/bpmn-io/dmnlint-pack-config/pull/1/commits/90b944637a42eafd4d365701e84058d66f680d0c).

File: .github/CI.yml (hunk 3 above)
Comment (codygulley) [reply .github/CI.yml]: Sorry about that that was definitely it. 
I see that it is now running on my fork 

Screen Shot 2022-08-23 at 2 22 34 AM

**Review by pinussilvestrus — APPROVED** [review]
Thanks a lot, looks good now, tests are green ✅

**Pull request merged**